src/js/components/List/List.js

```diff
@@ -262,18 +350,128 @@ const List = React.forwardRef(
                   };
                 }
 
+                let orderProps;
+                let orderControls;
+                if (onOrder) {
+                  orderProps = {
+                    draggable: true,
+                    onDragStart: event => {
+                      event.dataTransfer.setData('text/plain', '');
+                      // allowed per
+                      // https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API#define_the_drag_effect
+                      // eslint-disable-next-line no-param-reassign
+                      event.dataTransfer.effectAllowed = 'move';
+                      setDragging(index);
+                      setActive(undefined);
+                    },
+                    onDragEnd: () => {
+                      setDragging(undefined);
+                      setOrderingData(undefined);
+                    },
+                    onDragOver: event => {
+                      if (dragging !== undefined) {
+                        event.preventDefault();
+                        if (dragging !== index) {
+                          // eslint-disable-next-line no-param-reassign
+                          event.dataTransfer.dropEffect = 'move';
+                          setOrderingData(
+                            reorder(orderingData || data, dragging, index),
+                          );
+                          setDragging(index);
+                        }
+                      }
+                    },
+                    onDrop: () => {
+                      if (orderingData) {
+                        onOrder(orderingData);
+                      }
+                    },
+                    ref: dragging === index ? draggingRef : undefined,
+                  };
+
+                  const Up = theme.list.icons.up;
```

Comment:
This should be added to the docs under the theme section so users are aware they can change these.
`list.icons.up`